python_fundamentals-master/09_exceptions/09_05_check_for_ints.py

```diff
@@ -6,3 +6,11 @@
 The script should keep prompting the user until they enter an integer.
 
 '''
+
+try:
+    num_1 = int(input("please give me a number: "))
+    num_2 = int(input("please give me another number: "))
+except ValueError:
+    print("please enter a number and no letters")
+else:
+    print("good job!")
```

Comment:
This looks good, but it doesn't meet the requirements. It should keep asking until they do it.
So you need to do something about that 🤔